### ISSUE TITLE
HDMI video freezes shortly after startup (Atheros cards x 3)

### ISSUE THREAD

**Issue by htcohio** [DroneBridge/DroneBridge#30]
Hello,

Here is a short video showing how the video on the main display gradually degrades then freezes on the main hdmi out display.  The hotspot and usb tethering appears to be working fine now and the video continues to work perfectly on the android device. 

https://www.youtube.com/watch?v=Doq5FsyH3G4

**Comment by careyer** [comment]
@htcohio : does this only happen when a secondary display (USB tethering / hotspot) is connected or does the effect also show up when the HDMI is the only display used? Thanks!

**Comment by htcohio** [comment]
@careyer Hey, no it happens with or without the USB tethering or hotspot connected.  

Again, the video on the phone with USB tethering seems perfectly fine while video degrades on the main HDMI display. 

Last night I reflashed the image and double checked the settings and I had the exact same result with 1 or 2 atheros cards on the ground.



Have you happy to share another video or any other info that would help today.

I also used
